docs/docs/05_ipn/01_config-ipn.md

```
@@ -8,6 +8,16 @@ Khi thanh toán hoàn tất, VNPay sẽ gửi một thông báo IPN (Instant Pay
 Để cài đặt URL IPN cho môi trường production, bạn cần liên hệ trực tiếp với VNPay để được hỗ trợ.
 :::
 
+### Truy cập nhanh (Sau khi đăng nhập)
+
+Sau khi đăng nhập vào cổng thông tin merchant của VNPay, bạn có thể truy cập trực tiếp vào trang cấu hình Terminal (nơi cài đặt URL IPN) bằng cách truy cập:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)